data/mods/Xedra_Evolved/items/resources.json

```diff
@@ -56,7 +56,26 @@
     "name": { "str_sp": "water" },
     "copy-from": "water",
     "calories": 0,
-    "extend": { "flags": [ "NUTRIENT_OVERRIDE" ] }
+    "extend": { "flags": [ "NUTRIENT_OVERRIDE" ] },
+    "contamination": [
```

Comment:
Shouldn't this be extended instead of overriden?

Reply:
Done and tested.